installer/pyinstaller/build-linux.sh

```diff
@@ -49,7 +49,7 @@ mkdir -p .build/output/openssl
 cd .build/output/openssl
 
 echo "Building OpenSSL"
-curl "https://www.openssl.org/source/openssl-${openssl_version}.tar.gz" --output openssl.tar.gz
+curl -L "https://github.com:443/openssl/openssl/releases/download/OpenSSL_${openssl_version//./_}/openssl-${openssl_version}.tar.gz" --output openssl.tar.gz
```

Comment:
We can probably remove the port here. Not that I'd imagine Github would open their webserver on anything other than port 443, but its not required.